tools/crm_mon.c

```diff
@@ -2311,7 +2313,7 @@ mon_st_callback_display(stonith_t * st, stonith_event_t * e)
         /* disconnect cib as well and have everything reconnect */
         mon_cib_connection_destroy(NULL);
     } else {
-        out->progress(out);
+        out->progress(out, false);
```

Comment:
I'm thinking the crm_mon.c portion of this commit should move to the need_pass commit

Reply:
With all the end parameter stuff moved to the right commit, all that's left in crm_mon.c is adding cib__set_output.  I could move that to the need_pass commit too, or just leave it alone.  I'm not sold either way.

Reply:
Yeah now that I think about it, maybe just combining the two commits is better

Reply:
Done.